src/content/tutorials/fleet-management/cluster-management/migration-to-cluster-api/_index.md

```diff
@@ -56,6 +56,69 @@ Our engineers will check that all resources and infrastructure are correctly mig
 - Some customers have been using [k8s-initiator-app](https://github.com/giantswarm/k8s-initiator-app/) to configure some aspects of the workload cluster API.
   In the new Cluster API implementation, [most of the features enabled by the app](https://github.com/giantswarm/capi-migration-cli/tree/main/k8s-initiator-features) are now supported natively by the platform. The app should be removed and moved to the new syntax if our migration CLI doesn't handle your use-case. Giant Swarm account engineers will help you with this process.
 
+- [Service Account issuer switch](#service-account-issuer-switch)
+
+- [Cluster manifest clean-up](#cluster-manifest-clean-up)
+
+### Service Account issuer switch
+
+For context, a cluster Service Account issuer is an OIDC provider that signs and issues the tokens for the cluster's Service Accounts. In turn, these Service Accounts can be used to authenticate workloads against the Kubernetes API and against the AWS API via IRSA.
+
+In CAPA, since the cluster domain name changes from Vintage, a new Service Account issuer is also introduced. To make for a smooth migration though, we support defining multiple issuers in a cluster. This way, Service Account tokens issued by all the defined issuers will be accepted in the cluster's Kubernetes API. The accepted issuers are defined in `cluster.providerIntegration.controlPlane.kubeadmConfig.clusterConfiguration.apiServer.serviceAccountIssuers` in the cluster App values (order matters as we'll see below).
```

Comment:
```suggestion
In CAPA, since the cluster domain name changes from vintage, a new service account issuer has also been introduced. To make for a smooth migration though, we support defining multiple issuers in a cluster. This way, service account tokens issued by all the defined issuers will be accepted in the cluster's Kubernetes API. The accepted issuers are defined in `cluster.providerIntegration.controlPlane.kubeadmConfig.clusterConfiguration.apiServer.serviceAccountIssuers` in the cluster app values (order matters as we'll see below).
```